src/Crypt32.Shared/Crypt32+CRYPT_KEY_PROV_INFO.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace PInvoke
+{
+    using System;
+    using System.Runtime.InteropServices;
+
+    public partial class Crypt32
```

Comment:
Each partial declaration should contain a standardized xml comment for example : https://github.com/AArnott/pinvoke/blob/master/src/User32.Desktop/User32+SafeHookHandle.cs#L9